src/activity/activity.jsx

```diff
@@ -0,0 +1,141 @@
+import { h, Component } from 'preact'
+import cx from 'classnames'
+
+import * as _ from 'utils'
```

Comment:
or wait perhaps I answered too quickly here, perhaps some of the functions are lodash-like, in that case we can use them, they where at first tied to a specific datastrucutre, but then I simplified and we could indeed be using lodash ones.